person.rb

```diff
@@ -0,0 +1,28 @@
+class Person
+  attr_accessor :id, :name, :age
```

Comment:
- [ ] Per the project requirements, the `id` variable needs to only have getters set for it since we don't need to write the age after it has been set. We only need to be able to read it. So I will suggest you make use of the `attr_reader` instead for the `id` :wink: